### PR TITLE
Add Citrix Netscaler HA sensors and alerts

### PR DESCRIPTION
**Pull request by siegsters**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: misc/alert_rules.json [hunk 1]
```diff
@@ -351,6 +351,18 @@
   {
    "rule": "sensors.sensor_current ~ \"[5|6]\" && sensors.sensor_oid = \".1.3.6.1.4.1.674.10892.2.2.1\"",
    "name": "Dell iDRAC Global System Status Critical/NonRecoverable"
-   }
+   },
+   {
+    "rule": "devices.os = \"Netscaler\" && sensors.sensor_type = \"sysHighAvailabilityMode\" && sensors.sensor_current != `sensors.sensor_prev`",
```
Comment (laf) [inline misc/alert_rules.json]: Can you add `&& macros.device_up = 1` to each of these please.

**Comment by laf** [comment]
Thanks for doing the tests, you actually need to commit the snmprec file as well :)

**Comment by siegsters** [comment]
Sorry, for that. Added the netscaler.snmprec file

**Comment by laf** [comment]
I'd already tried that null removal :(

**Pull request merged**